tf/aks/README.md

```diff
@@ -0,0 +1,85 @@
+# Azure Kubernetes Service Module
+
+This module is responsible for the deployment of Azure Kubernetes Service inside a given environment. The module is developed in its own repository [here](https://github.com/venkatsunilm/terraform-azure-ref-aks-module).
+
+The [environment-base](../environment-base/README.md) deployment has to be run before this one.
+
+# Usage
+
+Create a service principal for Azure Kubernetes Service, following [this documentation](https://docs.microsoft.com/en-us/azure/aks/kubernetes-service-principal).
+
+or the quick way to follow these steps to achieve the goal. 
+
+1. az account show
+
+{
+  "environmentName": "AzureCloud",
+  "homeTenantId": "3cc81ecd06f86",
+  "id": "0cea2519ddb",
+  "isDefault": true,
+  "managedByTenants": [],
+  "name": "Azure subscription venkat test",
+  "state": "Enabled",
+  "tenantId": "3cc81ecd06f86",
+  "user": {
+    "name": "ve...@hotmail.com",
+    "type": "user"
+  }
+}
+
+
+2. Use the below command to create a RBAC
+az ad sp create-for-rbac --name ServicePrincipalName --role Contributor --scopes /subscriptions/0cea2519ddb/resourceGroups/tf-ref-dev-rg --sdk-auth
+
+{
+  "clientId": "c0107cd027c8",
+  "clientSecret": "wRb8QF-8pyav7",
+  "subscriptionId": "0cea2519ddb",
+  "tenantId": "3cc81ecd06f86",
+  "activeDirectoryEndpointUrl": "https://login.microsoftonline.com",
+  "resourceManagerEndpointUrl": "https://management.azure.com/",
+  "activeDirectoryGraphResourceId": "https://graph.windows.net/",
+  "sqlManagementEndpointUrl": "https://management.core.windows.net:8443/",
+  "galleryEndpointUrl": "https://gallery.azure.com/",
+  "managementEndpointUrl": "https://management.core.windows.net/"
+}
```

Comment:
It is crucial that you do not share your Azure SP credentials, including IDs and secrets, in any git files. This information should be kept private to ensure the security of your account and data. Please replace them with dummy values

Reply:
Sure again it's dummy values but then yes I understood it's better to replace with XXXXXXX. I will replace it. :)

Reply:
Replaced with dummy names and removed the unwanted info.